source/declarations.tex

```diff
@@ -5361,19 +5361,25 @@
 \keyword{char16_t} array,
 \keyword{char32_t} array,
 or \keyword{wchar_t} array
-can be initialized by
+may be initialized by
 an ordinary string literal,
 UTF-8 string literal,
 UTF-16 string literal,
 UTF-32 string literal, or
 wide string literal,
 respectively, or by an appropriately-typed \grammarterm{string-literal} enclosed in
 braces\iref{lex.string}.
+Additionally, an array of \keyword{char} or \keyword{unsigned} \keyword{char}
```

Comment:
```suggestion
Additionally, an array of \keyword{char} or \tcode{\keyword{unsigned} \keyword{char}}
```

Reply:
done, but why is this needed? I see that it's used elsewhere too.

Reply:
I suppose it's because it makes semantic sense. They're two adjacent keywords in the same piece of code. If you look at it from the generated PDF or HTML, the space in-between wouldn't be in monospace, and for the latter, not part of the same HTML code element.